src/BUILD.bazel

```diff
@@ -0,0 +1,1962 @@
+# @s2geometry/BUILD.bazel
+load("@rules_cc//cc:defs.bzl", "cc_library", "cc_test")
+package(default_visibility = ["//visibility:public"])
+
+cc_library(name = "s2-includes",includes = ["s2"],)
+
+cc_library(
+    name = "base",
```

Comment:
Can you explain the organization?  Why is this library here instead of in `src/s2/base/BUILD.bazel`?

Reply:
agreed